changelog.txt

```diff
@@ -1,3 +1,7 @@
+Version 11.0.3
```

Comment:
This is an "Added" so this should be a minor release (ie 11.1.0).

Reply:
Fixed